interfaces/Rule_options.atd

```diff
@@ -153,6 +153,12 @@ type t = {
   (* Whether a rule should be considered for interfile analysis. *)
   ~interfile <ocaml default="false"> : bool;
 
+  (* Whether last expression of a block is treated the same as a return statement.
+     This option is specific only to languages that treat the last statement of
```

Comment:
Is this only for the last statement? What the "last statement" means here exactly?

Reply:
I'll add better documentation. Basically I'm trying to say thata ny expression/statement that is the last to be evaluated  before returning from a function no matter what path it took. So there can be many "last statements" in a function.